.github/workflows/action-image-scan-deploy-dashing.yaml

```diff
@@ -0,0 +1,31 @@
+on: 
+  push:
+    branches:
+      - master
+      - main
+
+env:
+  ROS_VERSION: 2
+  UBUNTU_DISTRO: bionic
+  ROS_DISTRO: foxy
+
+jobs:
+  build-secure-and-push:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+
+    - run: docker build --build-arg ROS_VERSION=$ROS_VERSION --build-arg UBUNTU_DISTRO=$UBUNTU_DISTRO --build-arg ROS_DISTRO=$ROS_DISTRO --target devel-build -t ros$ROS_VERSION-$ROS_DISTRO:devel .
```

Comment:
I think I prefer to define the tag as `ros-$ROS_DISTRO:devel` to be compliant with the apt package names (e.g. `ros-foxy-urdf`) and the default install space `/opt/ros/foxy`

Reply:
fixed in 79e66f98b39febae6aaa5a1d60de4fa19dfdd956